pyaerocom/io/mscw_ctm/reader.py

```diff
@@ -171,16 +171,16 @@ class ReadMscwCtm:
 
     DEFAULT_FILE_NAME = "Base_day.nc"
 
+    YEAR_PATTERN = r".*((?:19|20)\d\d).*"
```

Comment:
Could you please add some explanation in the code about what this regex is doing?

Reply:
Added docs: https://github.com/metno/pyaerocom/blob/ea8d117ec29362b44760f1436c210bf17da96ccc/pyaerocom/io/mscw_ctm/reader.py#L174